scripts/publish-site.sh

```diff
@@ -11,6 +11,11 @@ yarn
 echo "=== Copying built Gatsby to website."
 gatsby-dev --scan-once --quiet
 
+echo "=== find graphql packages"
+
+find node_modules -name graphql
+echo "=== END find graphql packages"
```

Comment:
What use do you see to having graphql set as a peerDependency? I'm thinking about just removing it because plugins *have* to use the same version of GraphQL as Gatsby's core package.